extensions/interactions/CodeRepl/validator.js

```diff
@@ -20,10 +20,14 @@ oppia.factory('CodeReplValidationService', [
   'baseInteractionValidationService',
   function(baseInteractionValidationService) {
     return {
+      getCustomizationArgsWarnings: function(customizationArgs) {
```

Comment:
Would you mind adding a TODO here, or an issue to complete the implementation of this function? There really should be some.

Ditto for the other "blank" ones. (It's great that we're exposing them like this.)